ultravox/data/datasets.py

```diff
@@ -796,8 +802,11 @@ def __init__(self, args: VoiceDatasetArgs) -> None:
         )
         self._init_dataset(dataset)
 
-    def _get_sample(self, row) -> VoiceSample:
-        return self._get_transcribe_sample(row, tproc=text_proc.format_asr_text)
+    def _get_sample(self, row) -> Optional[VoiceSample]:
+        try:
+            return self._get_transcribe_sample(row, tproc=text_proc.format_asr_text)
+        except text_proc.FormatASRError:
+            return None
```

Comment:
It looks like all the `try..except` statements can be moved to `_get_transcribe_sample`?